### DISCUSSION TITLE
I love toast!

### DISCUSSION THREAD

**Discussion by DutchJelly** [timolins/react-hot-toast#185] [Show and tell]
Good evening everyone

I just wanted to leave a quick message. I love this library. It's so easy and intuitive to use, and really helps me a ton with error management. Normally I would use all kinds of state variables to show error messages.. but toast just handles it all! Oh.. and it somehow also does this while looking amazing.

Thanks to all contributors. I will try to contribute myself too.

Kind regards,
Jelle

**Comment by timolins** [comment]
Great to hear that you enjoy using react-hot-toast. Much appreciated 💖
